functions/exercise2/scriptSpec.js

```diff
@@ -1 +1,35 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals isPalindrome*/
+
+describe("Checking if word is palindrome: ", function () {
+    it("word 'tenet' is a palindrome", function () {
+        var palindrome = isPalindrome("tenet");
```

Comment:
variable name palindrome may be misleading as you are not sure if it is palindrome. isPalindrome name is better here and in all tests